test/WebSites/BasicWebSite/Startup.cs

```diff
@@ -28,6 +28,7 @@ public void ConfigureServices(IServiceCollection services)
                     // Filter that records a value in HttpContext.Items
                     options.Filters.Add(new TraceResourceFilter());
                 })
+                .SetCompatibilityVersion(CompatibilityVersion.Version_2_1)
```

Comment:
If the new line is needed, something is very wrong. The test fixture should be doing this for all test sites already. Changing the version explicitly is primarily useful _temporarily_ when manually testing a site or in the sandbox.